scripts/hardfork/run-localnet.sh

```diff
@@ -142,12 +142,12 @@ jq "$update_config_expr" > $CONF_DIR/base.json << EOF
 }
 EOF
 
-MAINNET_TO_BERKELEY_EXPR='.ledger.accounts = [.ledger.accounts[] | del(.token_permissions, .permissions.stake) | .token = "wSHV2S4qX9jFsLjQo8r1BsMLH2ZRKsZx6EJd1sbozGPieEC4Jf" | .token_symbol = "" | select(.permissions.set_verification_key == "signature").permissions.set_verification_key |= {auth:"signature", txn_version: "1"} ]'
+MAINNET_TO_DEVELOP_EXPR='.ledger.accounts = [.ledger.accounts[] | del(.token_permissions, .permissions.stake) | .token = "wSHV2S4qX9jFsLjQo8r1BsMLH2ZRKsZx6EJd1sbozGPieEC4Jf" | .token_symbol = "" | select(.permissions.set_verification_key == "signature").permissions.set_verification_key |= {auth:"signature", txn_version: "1"} ]'
```

Comment:
We should remove this conversion completely, because it's no longer relevant. We changed the format of accounts in Berkeley, but we are not going to change it in this release.